control-plane/tests/test_service_restore_drill_transactions.py

```
@@ -0,0 +1,497 @@
+            restore_summary.restore_drill.verified_reconciliation_ids,
+        )
+        self.assertEqual(restore_drill.verified_case_ids, (promoted_case.case_id,))
+
+        restored_case_detail = restored_service.inspect_case_detail(promoted_case.case_id)
+        self.assertEqual(restored_case_detail.linked_evidence_ids, (evidence_id,))
+
+    def test_service_phase21_restore_drill_falls_back_when_nested_transaction_blocks_repeatable_read(
+        self,
+    ) -> None:
+        base_store, _ = make_store()
+        _store, service, promoted_case, _evidence_id, _reviewed_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `service` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)